content/posts/scientific-python/dev-summit-2/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Developer Summit 2"
+date: 2024-09-19
+draft: false
+description: "Summary of the 2024 Scientific Python Developer summit."
+tags: ["Summit", "Scientific-Python"]
+displayInList: true
+author:
+  [
+    "Stéfan van der Walt",
+    "Jarrod Millman",
+    "Brigitta Sipőcz",
+    "Pamphile Roy",
+    "Matthew Feickert",
+    "Melissa Mendonça",
+  ]
+
+resources:
+  - name: featuredImage
+    src: "group.jpg"
+    params:
+      description: "Group picture of (most of the) summit attendees."
+      showOnTop: true
+
+summary: |
+  The 2024 Scientific Python Developer summit was held 3–5 June in Seattle. Here's a summary of what we did.
+---
+
+This post is a while overdue, but it's been a busy summer for everyone!
+
+In June, several of us got together for the "annual" (well, we've had it twice now) Scientific Python developer summit in Seattle. Our friends at the eScience Institute were again kind enough to host us. This time around, we made the event a bit shorter: a week is a long time to take off, and it also gets a bit overstimulating/exhausting, given the amount of conversations and work that take place.
```

Comment:
```suggestion
In June, several of us got together for the "annual" (well, we've had it twice now) Scientific Python developer summit in Seattle. Our friends at the eScience Institute were again kind enough to host us. This time around, we made the event a bit shorter: a week is a long time to take off, and it also gets a bit overstimulating/exhausting, given the amount of work that takes place.
```

Reply:
How about: "given the amount of conversation and work that takes place.".  It's a bit of a cheat, but I think we can get away with it.

Reply:
Yeah.
```suggestion
In June, several of us got together for the "annual" (well, we've had it twice now) Scientific Python developer summit in Seattle. Our friends at the eScience Institute were again kind enough to host us. This time around, we made the event a bit shorter: a week is a long time to take off, and it also gets a bit overstimulating/exhausting, given the amount of conversation and work that takes place.
```